update/sync/coordinator.go

```diff
@@ -101,49 +112,65 @@ func (ss *syncState) SyncAllState(epoch uint32) error {
 	go func() {
 		defer wg.Done()
 
+		ctxDisplay, cancelDisplay = context.WithCancel(context.Background())
+		go displayStatusMessage(fmt.Sprintf("syncing pending miniblocks for epoch %d", epoch), ctxDisplay)
+
 		ctx, cancel := context.WithTimeout(context.Background(), time.Hour)
 		errSync := ss.miniBlocks.SyncPendingMiniBlocksFromMeta(meta, unFinished, ctx)
+		cancelDisplay()
 		cancel()
 		if errSync != nil {
 			mutErr.Lock()
-			errFound = errSync
+			errFound = fmt.Errorf("%w in syncState.SyncAllState - SyncPendingMiniBlocksFromMeta", errSync)
 			mutErr.Unlock()
 			return
 		}
 
+		ctxDisplay, cancelDisplay = context.WithCancel(context.Background())
+		go displayStatusMessage("syncing miniblocks", ctxDisplay)
 		syncedMiniBlocks, errGet := ss.miniBlocks.GetMiniBlocks()
+		cancelDisplay()
 		if errGet != nil {
 			mutErr.Lock()
-			errFound = errGet
+			errFound = fmt.Errorf("%w in syncState.SyncAllState - GetMiniBlocks", errGet)
 			mutErr.Unlock()
 			return
 		}
 
+		ctxDisplay, cancelDisplay = context.WithCancel(context.Background())
+		go displayStatusMessage(fmt.Sprintf("syncing pending transactions for epoch %d", epoch), ctxDisplay)
 		ctx, cancel = context.WithTimeout(context.Background(), time.Hour)
 		errSync = ss.transactions.SyncPendingTransactionsFor(syncedMiniBlocks, ss.syncingEpoch, ctx)
 		cancel()
+		cancelDisplay()
```

Comment:
cancelDisplay is done before cancel in line 120-121. Does it matter?

Reply:
Not each Sync... calls from this method has cancel method called afterwards

Reply:
refactored the order. However, the missing cancel() comes from the fact that the `GetMiniBlocks` function does not require a context